packages/react/src/primitives/Select/Select.tsx

```diff
@@ -23,11 +24,16 @@ export const Select: React.FC<SelectProps> = (props) => {
     ...rest
   } = props;
   return (
-    <View className="amplify-select-wrapper">
+    <View className={ComponentClassNames.SelectWrapper}>
       <View
+        aria-invalid={hasError}
         as="select"
         value={value}
-        defaultValue={defaultValue}
+        defaultValue={
```

Comment:
`value === undefined` is to get rid of controlled component. React will complain if we set both `defaultValue` and `value`